aten/src/ATen/native/cpu/DistributionTemplates.h

```diff
@@ -178,9 +179,9 @@ void normal_kernel(Tensor& self, double mean, double std, RNG generator) {
     normal_fill(self, static_cast<float>(mean), static_cast<float>(std), generator);
 #endif
   } else {
-    // half and bfloat16 cannot be properly tested due to the lack of other operations 
-    // like add/sub/mean implemented for half and bfloat16.
-    AT_DISPATCH_FLOATING_TYPES(self.scalar_type(), "normal_kernel_cpu", [&] {
+    // bfloat16 cannot be properly tested due to the lack of other operations 
```

Comment:
I think that `torch.half` should be added to `def test_normal` in test_torch.py

Reply:
Will do in a separate PR, although I'm a bit concerned about the perf implication.